app/assets/stylesheets/trips.css.scss

```diff
@@ -51,7 +51,11 @@
   position: relative;
   border: solid 3px #363636;
   border-radius: 5px 5px 5px 5px;
-  margin: 15px 15px 15px 15px
+  margin: 15px 15px 15px 15px;
+  box-shadow: 0px 30px 40px ;
+  -moz-box-shadow:    0px 0px 40px 15px rgba(0, 0, 0, 0.6);
+  -webkit-box-shadow: 0px 0px 40px 15px rgba(0, 0, 0, 0.6);
+  box-shadow:         0px 0px 40px 15px rgba(0, 0, 0, 0.6);
```

Comment:
Property `box-shadow` already defined on line 55<br>Colon after property should be followed by one space<br>`0px` should be written without units as `0`